docs/main/user-guides/local-storage-user-guide/local-pv-lvm/faq.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Are the below contents repeated in the master faq?

Reply:
No

Reply:
Hmm, not sot sure, I think there is a repetition. I see a question like "How to add custom topology key in the Local PV LVM driver?" in the master faq as well. Please check once again.

Reply:
I created a separate FAQ page for LVM and ZFS in their respective folders initially, later I moved all the FAQs into the master FAQ. So there is only one now. No repetition.